NAMESPACE

```diff
@@ -157,3 +157,5 @@ importFrom(utils,relist)
 importFrom(utils,tail)
 importFrom(xml2,as_list)
 importFrom(xml2,xml_find_all)
+importFrom(xml2,xml_find_first)
```

Comment:
how did this end up in the diff?

Reply:
intentional, note the corresponding diff in lintr-package.R.

Just to avoid merge conflicts / possibly forgetting to remove the :: on merge later, since they're used in the diff here.